File: MonoLRC/BrowserDetailActivity.cs
using Android.App;
using Android.OS;
using Android.Views;

namespace Monolrc
{
	[Activity (Label = "BrowserDetailActivity")]			
	public class BrowserDetailActivity : Activity
	{
		protected override void OnCreate (Bundle bundle)
		{
			base.OnCreate (bundle);
			
			ActionBar.SetDisplayHomeAsUpEnabled (true);
			
			if (bundle == null) {
				var details = new BrowserDetailFragment();
				details.Arguments = Intent.Extras;
				FragmentTransaction transaction = FragmentManager.BeginTransaction();
				transaction.Add(Android.Resource.Id.Content, details).Commit();
			}
		}
		
		public override bool OnOptionsItemSelected (IMenuItem item)
		{
			if (item.ItemId == Android.Resource.Id.Home)
			{
				Finish();
			}
			return base.OnOptionsItemSelected (item);
		}
	}
}

File: CollegeSportsFlavor/app/src/main/java/com/inoles/collegesports/BrowserDetailActivity.java
/*
 * Copyright (C) 2013 Jonathan Steele
 *
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 * You should have received a copy of the GNU General Public License
 * along with this program. If not, see <http://www.gnu.org/licenses/>.
 */

package com.inoles.collegesports;

import android.app.Activity;
import android.app.FragmentManager;
import android.os.Bundle;

public class BrowserDetailActivity extends Activity {
    // Called when the activity is first created.
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        FragmentManager fm = getFragmentManager();
        if (fm.findFragmentById(android.R.id.content) == null) {
            BrowserDetailFragment pane = new BrowserDetailFragment();
            pane.setArguments(getIntent().getExtras());
            fm.beginTransaction().add(android.R.id.content, pane).commit();
        }
    }
}
